junit-platform-surefire-provider/src/main/java/org/junit/platform/surefire/provider/JUnitPlatformProvider.java

```diff
@@ -134,15 +134,11 @@ private void invokeSingleClass(Class<?> testClass, RunListener runListener) {
 
 		Optional<List<String>> includes = getGroupsOrTags(getPropertiesList(INCLUDE_GROUPS),
 			getPropertiesList(INCLUDE_TAGS));
-		if (includes.isPresent()) {
-			filters.add(TagFilter.includeTags(includes.get()));
-		}
+		includes.ifPresent(includeFilters -> filters.add(TagFilter.includeTags(includeFilters)));
```

Comment:
@smoyer64, maybe I'm missing something, but it doesn't appear that `includeFilters` is from you. 😉